run_dir/static/js/base_b5.js

```diff
@@ -15,7 +15,10 @@ $(document).ready(function(){
   // Activate bootstrap hover tooltips
   var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-toggle="tooltip"]'))
   var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
-    return new bootstrap.Tooltip(tooltipTriggerEl)
+    return new bootstrap.Tooltip(tooltipTriggerEl, {
```

Comment:
The hovering (text turns black) only works for the running note buttons for categories "User communication", "Deviation", and "Important." Probably because these have custom color classes?

Reply:
Done!